src/Octoshift/Services/ReclaimService.cs

```diff
@@ -221,22 +229,30 @@ private bool HandleReclaimationResult(string mannequinUser, string targetUser, M
     {
         if (result.Errors != null)
         {
-            _log.LogError($"Failed to reclaim {mannequinUser} ({mannequin.Id}) to {targetUser} ({targetUserId}): {result.Errors[0].Message}");
-            return false;
+            // Writing as switch statement in anticipation of other errors that will need specific logic
+            switch (result.Errors[0].Message)
+            {
+                case string a when a.Contains("is not an Enterprise Managed Users (EMU) organization"):
+                    _log.LogError("Failed to reclaim mannequins. The --skip-invitation flag is only available to EMU organizations.");
+                    return false; // Indicates we should stop parsing through the CSV
+                default:
+                    _log.LogWarning($"Failed to reclaim {mannequinUser} ({mannequin.Id}) to {targetUser} ({targetUserId}): {result.Errors[0].Message}");
+                    return true;
+            }
```

Comment:
Is there a way that we can handle this error without parsing human-readable error messages?  Does the GraphQL response happen to include a key or something?  If so, we should watch for that :ok_hand: 

If not (and this is difficult to consume as an API user), we should probably file an issue on our end to make this easier to handle :slightly_smiling_face:

Reply:
@synthead At this point, I don't think so. 

This is where the error gets raised: https://github.com/github/github/blob/4ad31aba4642a3776068727dd578ca76e61911df/lib/platform/mutations/reattribute_mannequin_to_user.rb#L33

Is there an example of a key somewhere so I can see what you're envisioning?